exp/api/v1beta2/awsmanagedmachinepool_webhook.go

```diff
@@ -123,6 +123,10 @@ func (r *AWSManagedMachinePool) validateLaunchTemplate() field.ErrorList {
 		return allErrs
 	}
 
+	if r.Spec.InstanceTypes != nil {
```

Comment:
We can also log in the Create about the deprecation of the InstanceType field

Reply:
Added log line.